code/BTCUtil.c

```diff
@@ -780,6 +780,17 @@ Datas ckdIndicesFromPath(const char *cStr)
     return DTPopDatas(result);
 }
 
+Data privKeyToHdWallet(Data privKey, const char *passphrase)
+{
```

Comment:
Let's add a check here for `if(toMnemonic(privKey).bytes == NULL)` and if it is null return `DTPopNull();`